src/components/callout/styles.scss

```diff
@@ -28,30 +28,46 @@
     margin: 0;
   }
 
-  .alert-header {
+  .callout-header {
     font-weight: 500;
     color: inherit;
+
+    & a {
+      color: inherit;
+    }
+
+    &[role="button"] {
+      cursor: pointer;
+    }
   }
 }
 
-.alert-icon {
+.callout-icon {
   flex: 1em 0 0;
   height: 1.75em;
-  color: var(--alert-highlight-color);
+  color: var(--callout-highlight-color);
+
+  &[role="button"] {
+    cursor: pointer;
+  }
+}
+
+.callout-content {
+  min-width: 0;
```

Comment:
Without this, e.g. code boxes can lead to exceeding the width of the content. this should "fix" this.